flow/util/plot_floorplan.py

```diff
@@ -0,0 +1,203 @@
+import os
+import matplotlib.pyplot as plt
+from math import log
+
+#
+# Helper function to visualize the rtlmp macro placement
+# If run in ORFS rtlmp creates a dir in the objext/<platform>/<test_case>/<flow_variant>/rtlmp
+# link that dir to ./rtlmp
+# run python3 utils/plot_floorplan.py
```

Comment:
Doesn't the GUI plot macros already? Could we re-use GUI features and Tcl script to achieve this?

Reply:
This plot has additional information such as cluster sizes and and the bundled connection between clusters. Our goal is to enable this in the GUI. It is more of a debugging aid for the macro placer